llvm/test/CodeGen/AMDGPU/mad_64_32.ll

```diff
@@ -1333,5 +1333,90 @@ define i48 @mad_i48_i48(i48 %arg0, i48 %arg1, i48 %arg2) #0 {
   ret i48 %a
 }
 
+define i64 @lshr_mad_i64(ptr addrspace(1) %1) local_unnamed_addr #0 {
```

Comment:
Please pre-commit the tests so we can see what effect your patch has on the codegen.

Reply:
https://github.com/llvm/llvm-project/pull/119509